### ISSUE TITLE
Project64 3.0.1 crashes

### ISSUE THREAD

**Issue by AlexN64gamer** [ata4/angrylion-rdp-plus#136]
The emulator crashes with every game when I set the graphics settings to soft (Bilinear+Nearest Neightor), I still can hear sound but theres not picture on the screen, specs: Intel I7 9700K (8-cores) Nvidia RTX 2060 6Gb, 32gb ram Corsar Vengeance ddr5, SSD Corsair MP600 1Tb, im using slow mode, Unfiltered Output and Multithread is on

**Comment by Squall-Leonhart** [comment]
Known issue, use 4.0 nightly

**Comment by RokkumanX** [comment]
Alternatively use Rosalie241's equivalent to Project64, RMG instead if you want to take Mupen64Plus for a spin or if you are feeling really hardcore you can just use mupen64plus-ui-console which is what I use.

Pros for using Mupen64Plus:

- 64-bit (known to have issues on Project64)
- angrylion-rdp-plus working as it should, haven't had any issues for quite some time
- ParaLLEl RDP and RSP support (something Project64 lacks)

Cons for using Mupen64Plus:

- You'll have to configure some things via the .cfg file since RMG is still in development and not as fully developed as Project64 and if you are using mupen64plus-ui-console like I am for the most part configuring Mupen via the .cfg file is crucial since it's CLI only
- Rough around the edges sometimes but works none the less

No offense or criticism to Project64 but while it states that the emulator is free there is no denying that there is plenty of support nag screens showing up.

Just wanted to chime in and tell you that there are alternatives, at the end of the day I just want to play my games without to much hassle and Mupen64Plus does the job for me with grace.

**Comment by Jj0YzL5nvJ** [comment]
> ParaLLEl RDP and RSP support (something Project64 lacks)

[Coff](https://github.com/Rosalie241/pj64-parallelrdp/releases)... [coff](https://github.com/Rosalie241/parallel-rsp/releases)...

**Comment by RokkumanX** [comment]
What rock have I been living under?

**Comment by AlexN64gamer** [comment]
Sorry for opening this issue with this missleading title and without enough information, Im talking about Jet Force Gemini one of the games for the N64 that are hard to emulate (Rare Ltd. games like DK64, CBFD, Perfect Dark, etc) the issue im having with the last dev release of angrylyion rdp plus, is that, in the intro cutscene, as soon as Vella get to watch the red alert screen on the ship monitor, the game crashes, exactly at that point, Ive also happen to find other random crashes in-game but those do not give any specific error type I can put here, I just want you guys to know this and hopefully be fixed for 1.7 (if you dont have to much work)  thanks in advance and stay safe everyone, btw im using PJ64 latest dev version, Ive already wrotte a post in their github, the problem is not with the emulator because using any other video plugin, fixes the crashes, but the problem is that, the game doesnt look as accurate as it does with angrylion

**Comment by Squall-Leonhart** [comment]
Issues caused and within the angry lion plugin will not be fixed by the project64 developers.

**Comment by Jj0YzL5nvJ** [comment]
Stable versions are a suggestion.

In the development channels, no one cares about "deprecated stable releases", only the ones in development matter. If the problem does not exist in the main development branch, then the report is invalid.

**Comment by AlexN64gamer** [comment]
> Issues caused and within the angry lion plugin will not be fixed by the project64 developers.

Yep, I know, I didnt ask them to fix the issue, am asking angrylion´s team

**Comment by AlexN64gamer** [comment]
Im using everything you suggested, there isnt any change, this game is unplayable with the latest version, crashes when entering a door..... crashes in cutscenes.... etc

**Comment by AlexN64gamer** [comment]
Ok, update; using cpu on interpreter and uncheking "protect memory" works for now, I havent had any crashes in cutscenes or going into doors, or even starting a saved game (like before), im using the dev latest release (1.6-17-g6660e93) x86, without Paralell RSP (just the PJ64 default one) PJ64 version is the latest nightly build, so it is working... but, shouldnt it work with protect memory and cpu recompiler?  @Jj0YzL5nvJ

**Comment by Jj0YzL5nvJ** [comment]
It works with:
<img src="https://i.imgur.com/2pRUM8H.png" alt="" class="">
I didn't need an external RSP. Credits for @ReyVGM - project64/project64#1773

**Comment by RokkumanX** [comment]
Any news @AlexN64gamer? Is the issue still relevant as of this day?
